tests/cms/test_event_filters.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from urllib.parse import urlencode
+
+import pytest
+from django.test.client import Client
+from django.urls import reverse
+
+
+@pytest.mark.django_db
+def test_event_filters(load_test_data: None, admin_client: Client) -> None:
```

Comment:
I didn't find any way to test the event filter by `view_config.py`.
If this looks like the right way to go, I'll add other test cases and the documentation.

Reply:
Maybe it's stupid idea, but a quick way to add get parameters to the existing view tests would be something like converting the view name to an optional tuple, e.g. instead of:
https://github.com/digitalfabrik/integreat-cms/blob/6e60e28067c1f525019e12e6e8671060e9784c46/tests/cms/views/view_config.py#L266

we write
```
(
    (
        "events",
        parse.urlencode({
            "events_time_range": "CUSTOM",
            "date_from": "2023-01-01",
            "date_to": "2030-12-31",
            "poi_id": -1,
            "all_day": 1,
            "recurring": 2,
        })
    ),
    ROLES
)
```

And then in the test utils, we can change:
https://github.com/digitalfabrik/integreat-cms/blob/6e60e28067c1f525019e12e6e8671060e9784c46/tests/cms/views/utils.py#L37

to:
```
if isinstance(view_name, tuple):
    url = reverse(view_name[0], kwargs=kwargs) + "?" + view_name[1]
else:
    url = reverse(view_name, kwargs=kwargs)
```

But I didn't test this out. This also makes the view config even more complex and also requires changes to the type annotations.